tests/test_code_gen.py

```diff
@@ -611,6 +611,24 @@ def test_annassign(self):
         """
         self.assertAstRoundtripsGtVer(source, (3, 6))
 
+    @unittest.skipUnless(sys.version_info >= (3, 6),
+                         "typing and ann assignment was introduced in "
+                         "Python 3.6")
+    def test_function_typing(self):
+        source = """
+        def foo(x : int) ->str:
+            i : str = '3'
+            return i
+        """
+        srctxt = canonical(source)
+        target = """
+        def foo(x: int) -> str:
+            i: str = '3'
+            return i
+        """
+        trgtxt = canonical(target)
+        self.assertEqual(self.to_source(ast.parse(srctxt)).rstrip(), trgtxt)
```

Comment:
More importantly, can't we directly use `assertAstEqualsSource` here?

Reply:
I have used `assertAstEqualSource` in rvprasad/astor@78e7c60a0.